src/controller/user.controller.ts

```diff
@@ -3,13 +3,59 @@ import {
   createTagsIfNotExists,
   findTagIdsByTagName,
 } from "@/repository/tag.repository";
-import { createUser } from "@/repository/user.repository";
 import {
+  createUser,
+  insertProfileImage,
+  isExistUser,
+} from "@/repository/user.repository";
+import {
+  EXISTS_INITIAL_INFORMATION_MESSAGE,
+  NO_EXISTS_INITIAL_INFORMATION_MESSAGE,
   PROFILE_CREATE_SUCCESS,
+  PROFILE_IMAGE_UPDATE,
   SERVER_INTERNAL_ERROR_MESSAGE,
 } from "@/constants/message";
 import { string } from "prop-types";
 import { NextApiRequest, NextApiResponse } from "next";
+import { InitialInformationRequestBody } from "@/models/user/InitialInformationRequestBody";
+import multer from "multer";
+import { multipartUploader } from "@/service/imageUploader";
+import * as path from "path";
+import { auth } from "@/service/firebase";
+import { uuidv4 } from "@firebase/util";
+
+interface response {
+  exists: boolean;
+  message: string;
+}
+
+export const getUser = async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
회원이 있느냐 없느냐를 반환하기 때문에 아래와같은 함수 이름이 더 적절해보이네요
```suggestion
export const isUserExists = async (req: NextApiRequest, res: NextApiResponse) => {
```

Reply:
user.repository와 충돌이 날 것 같습니다.

Reply:
@Antonio-990510 import할때 구분해서 쓰면되긴하는데,,, 음 아니면 내부의 함수들 모두를 클래스로 감싸야할것같네요

Reply:
import 할때 구분해서 쓰면 될것같네요~! 매개변수가 다르니 크게 문제될것같진 않습니다.